front/pages/api/stripe/webhook.ts

```diff
@@ -171,6 +171,36 @@ async function handler(
           // The stripe subscription becomes "past_due".
           // We keep active and email the user and us to manually manage those cases first?
           break;
+        case "customer.subscription.updated":
+          // Occurs when the subscription is updated:
+          // - when the number of seats changes for a metered billing.
+          // - when the subscription is canceled by the user: it is ended at the of the billing period, and we will receive a "customer.subscription.deleted" event.
+          // - when the subscription is activated again after being canceled but before the end of the billing period.
+          break;
+        case "customer.subscription.deleted":
+          // Occurs when the subscription is canceled by the user or by us.
+          const stripeSubscription = event.data.object as Stripe.Subscription;
+          if (stripeSubscription.status === "canceled") {
```

Comment:
FMI, in which case would we receive this webhook but not have the cancelled status ?
Also, I always thought it was `cancelled`  but looks like americans use `canceled` 😳  TIL

Reply:
If we get the webhook but not the status, should it be an error ?

Reply:
Actually they are also unsure about the single or two 'l' 😄 , this is the content of the event: 
<img width="773" alt="Capture d’écran 2023-11-02 à 11 48 31" src="https://github.com/dust-tt/dust/assets/3803406/a0bfb8ef-1d3e-4b55-a0c3-832bab23b10b">